torch/fx/node.py

```diff
@@ -87,6 +88,12 @@ def _get_qualified_name(func: Callable[..., Any]) -> str:
        and func is getattr(torch.Tensor, func.__name__, None)):
         return f"torch.Tensor.{func.__name__}"
     name = func.__name__
+    if name == "<lambda>":
+        # For lambdas, try to get their defining name in the module
+        try:
+            name = inspect.getsource(func).split("=")[0].strip()
+        except Exception:
+            raise RuntimeError("Unable to represent lambda")
```

Comment:
@Chillee could you look at this?

Reply:
In particular, I'm wondering if the name we generate here must actually be "dereferenceable" (e.g., eval(_get_qualified_name(func)) is func`